src/ucp/api/ucp.h

```diff
@@ -2502,6 +2503,53 @@ ucs_status_t ucp_put(ucp_ep_h ep, const void *buffer, size_t length,
 ucs_status_t ucp_put_nbi(ucp_ep_h ep, const void *buffer, size_t length,
                          uint64_t remote_addr, ucp_rkey_h rkey);
 
+/**
+ * @ingroup UCP_COMM
+ * @brief Non-blocking implicit remote memory put operation.
```

Comment:
This is a non-blocking put operation, not a non-blocking implicit operation, right?

Reply:
No idea. C&P'ed text. What is the definition of implicit in UCX?

Reply:
Implicit means - no request/cb is returned. Good catch. it is non-blocking operation.

Reply:
Ah, ok. Will fix the wording.